tox.ini

```diff
@@ -105,7 +104,7 @@ commands =
     checkformatting: black --check h tests bin
     checkformatting: isort --quiet --check-only h tests bin
     {tests,functests}: sh bin/create-testdb
-    tests: coverage run -m pytest {posargs:tests/h/}
+    tests: python -m pytest --cov --numprocesses logical --dist loadgroup {posargs:tests/h/}
```

Comment:
The `--dist loadgroup` is what enables the grouping thing that we're using to avoid running Elasticsearch tests in parallel.